README.md

```diff
@@ -15,6 +15,102 @@ Our aim is to infuse these strengths into the Ethereum consensus client ecosyste
+- **Videos**:
+  - [Basic technical details of Ethereum](https://youtu.be/gjwr-7PgpN8)
+  - [Ethereum in 30 minutes](https://youtu.be/UihMqcj-cqc)
+  - [Ethereum Foundation youtube channel](https://www.youtube.com/@EthereumFoundation)
+  - [Ethereum youtube channel](https://www.youtube.com/@EthereumProtocol)
+- **Blogs**:
+  - [Vitalik Buterin's blog](https://vitalik.ca/)
+  - [Ethereum Foundation blog](https://blog.ethereum.org/)
+  - [Ethereum Magicians forum](https://ethereum-magicians.org/)
+  - [Ethresear.ch forum](https://ethresear.ch/)
+  - [EIP's](https://eips.ethereum.org/)
+  - [ACD & Related meetings](https://github.com/ethereum/pm)
+- **Specifications**:
+  - [Consensus specs](https://github.com/ethereum/consensus-specs)
+  - [Vitalik Buterin's annotated specs](https://github.com/ethereum/annotated-spec)
+  - [Eth2Book annotated specs](https://eth2book.info/capella/part3/)
+
+While some of the resources listed might appear outdated, it's important to understand that the Ethereum protocol is continuously evolving. As such, there isn't a definitive, unchanging source of information. However, these resources, even if older, provide foundational knowledge that remains pertinent to understanding the protocol's core concepts.
+
+Truly mastering the Ethereum protocol is a complex endeavor. The list provided here is just a starting point, and delving deeper will necessitate exploring a broader range of readings and resources. As you immerse yourself in the project, continuous learning and adaptation will be key.
+
+If you come across any resource that you find invaluable and believe should be added to this list, please don't hesitate to suggest its inclusion.
+
+#### 3. **Dive In**:
+
+With your newfound knowledge, explore the various areas of our project. Whether you're interested in the core consensus layer, networking, CLI, documentation, testing, or tooling, there's a place for you.
+
+Start by browsing our [issues](https://github.com/lambdaclass/lambda_ethereum_consensus/issues), especially those tagged as `good first issue`. These are beginner-friendly and a great way to familiarize yourself with our codebase.
+
+### Contributing:
+
+Found an issue you're passionate about? Comment with `"I'd like to tackle this!"` to claim it. Once assigned, you can begin your work. After completing your contribution, submit a pull request for review. Our team and other contributors will be able to provide feedback, and once approved, your contribution will be merged.
+
+Please adhere to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification when crafting commit messages.
+
+### Communication:
+
+**For specific questions** related to issues, you can use the chat provided under each issue.
+
+**For general questions**, discussions, and feedback, you can use the [Lambda Consensus Client](https://t.me/lambdaconsensus) telegram group.
+
+### Together, let's shape the future of Ethereum! 🚀
```

Comment:
nice comment, but I would change it to not be a headline, maybe bold instead?

Reply:
done